frontend/ui/elements/screensaver_menu.lua

```diff
@@ -26,114 +26,140 @@ local function genMenuItem(text, setting, value, enabled_func, separator)
         separator = separator,
     }
 end
-
 return {
-    genMenuItem(_("Use last book's cover as screensaver"), "screensaver_type", "cover", hasLastFile),
-    genMenuItem(_("Use book status as screensaver"), "screensaver_type", "bookstatus", hasLastFile),
-    genMenuItem(_("Use random image from folder as screensaver"), "screensaver_type", "random_image"),
-    genMenuItem(_("Use document cover as screensaver"), "screensaver_type", "document_cover"),
-    genMenuItem(_("Use image as screensaver"), "screensaver_type", "image_file"),
-    genMenuItem(_("Use reading progress as screensaver"), "screensaver_type", "readingprogress", isReaderProgressEnabled),
-    genMenuItem(_("Leave screen as-is"), "screensaver_type", "disable", nil, true),
-    -- separator
-    {
-        text = _("Add message to screensaver"),
-        checked_func = function()
-            return G_reader_settings:isTrue("screensaver_show_message")
-        end,
-        callback = function()
-            G_reader_settings:toggle("screensaver_show_message")
-        end,
-        separator = true,
-    },
-    -- separator
     {
-        text = _("Settings"),
+        text = _("Screensaver"),
         sub_item_table = {
+            genMenuItem(_("Show book cover on sleep screen"), "screensaver_type", "cover", hasLastFile),
+            genMenuItem(_("Show custom image on sleep screen"), "screensaver_type", "image_file"),
+            genMenuItem(_("Shuffle images on sleep screen"), "screensaver_type", "random_image"),
+            genMenuItem(_("Show document cover on sleep screen"), "screensaver_type", "document_cover"),
+            genMenuItem(_("Show reading progress on sleep screen"), "screensaver_type", "readingprogress", isReaderProgressEnabled),
+            genMenuItem(_("Show book status on sleep screen"), "screensaver_type", "bookstatus", hasLastFile),
+            genMenuItem(_("Lock the screen in current state"), "screensaver_type", "disable", nil, true),
+            separator = true,
             {
-                text = _("Screensaver folder"),
-                keep_menu_open = true,
-                callback = function()
-                    Screensaver:chooseFolder()
+                text = _("Border fill"),
+                enabled_func = function()
+                    return G_reader_settings:readSetting("screensaver_type") == "cover" or G_reader_settings:readSetting("screensaver_type") == "random_image"
+                    or G_reader_settings:readSetting("screensaver_type") == "image_file" or G_reader_settings:readSetting("screensaver_type") == "document_cover" 
```

Comment:
@poire-z naughty or nice?

Reply:
Naughty, to me :).
I would indent the 2nd line until the `or ` or the `G_reader` vertically align with the `G_reader` above.